### PR TITLE
fix: don't fail codegen on conflicts

### PR DESCRIPTION
**Pull request by crenshaw-dev**
The swagger CLI folks added a flag to ignore conflicts. This change will simplify the dev workflow.

### CODE REVIEW

**Review by blakepettersson — APPROVED** [review]
If this means that `make codegen` won't crap out after ~40 mins it has my seal of approval 👍 🚀

**Comment by toyamagu-2021** [comment]
I confirm this change works when we should increase `EXPECTED_COLLISION_COUNT`.

**Review by drpaneas — APPROVED** [review]

**Review by jannfis — APPROVED** [review]
Wohoo :tada:

**Pull request merged**
